liquibase-standard/src/main/java/liquibase/datatype/core/UUIDType.java

```diff
@@ -10,7 +10,7 @@
 
 import java.util.Locale;
 
-@DataTypeInfo(name = "uuid", aliases = { "uniqueidentifier", "java.util.UUID" }, minParameters = 0, maxParameters = 0, priority = LiquibaseDataType.PRIORITY_DEFAULT)
+@DataTypeInfo(name = "uuid", aliases = { "binary(16)", "binary", "uniqueidentifier", "java.util.UUID" }, minParameters = 0, maxParameters = 0, priority = LiquibaseDataType.PRIORITY_DATABASE)
```

Comment:
Right now binary is defined in BlobType  - this may cause issues for other databases :/
Maybe you need  to replicate some of this logic around `BlobType:73` ?

Reply:
Hmm, that is a big change for the tests. Should have ran them locally before pinging you.

I'm not sure if that line in BlobType will help any here, it seems to be a priority coming from the registry located here: `liquibase/datatype/DataTypeFactory.java:184`. There seems to be a datatype priority available in the `DataTypeInfo` attribute, but it's the same across the board (and blows up a lot of tests when you modify it from default). 

There needs to be a priority for a type, for a given database I think. Without the priority change, this just about works but the registry pulls blob, and uuid (in that order) and then just returns on blob because mysql supports it, and it's at index 0 for the registry classes. 

I went ahead and tried to implement a database priority, let me know what you think.

I also removed a test for mapping "BINARY(16)" -> BlobType for MySQL, I'm not sure if anyone would use that column type for anything else in MySQL?